src/neo/Network/P2P/RemoteNode.cs

```diff
@@ -124,6 +125,13 @@ protected override void OnReceive(object message)
                     RefreshPendingKnownHashes();
                     break;
                 case Message msg:
+                    if (msg.Payload is PingPayload payload)
+                    {
+                        if (payload.LastBlockIndex > LastHeightSent)
+                            LastHeightSent = payload.LastBlockIndex;
+                        else if (msg.Command == MessageCommand.Ping)
```

Comment:
But I mean,@shargon, if `Pong` has a height lower than `LastHeightSent` it will be processed anyway, right?

Reply:
Yes, because we should provide a response for the ping

Reply:
`Pong` already is the response.

Reply:
Yes, so we need to send it

Reply:
That is true...ajajja
Thanks